### PR TITLE
Add ea repo to Grape repos.

### PR DESCRIPTION
**Pull request by chirino**

### CODE REVIEW

**Comment by chirino** [comment]
Grape is taking a long time to download dependencies on some boxes.

**Comment by chirino** [comment]
[merge]

File: dao/src/main/resources/com/redhat/ipaas/dao/deployment.json [hunk 1]
```diff
@@ -450,7 +450,7 @@
...
...ions\": { \"fieldAction\": [] } }, \"outputField\": { \"jsonType\": \"com.mediadriver.atlas.v2.MappedField\", \"field\": { \"jsonType\": \"com.mediadriver.atlas.java.v2.JavaField\", \"name\": \"Description\", \"path\": \"description\", \"type\": \"STRING\", \"setMethod\": \"setDescription\" }, \"fieldActions\": { \"fieldAction\": [] } } }, { \"jsonType\": \"com.mediadriver.atlas.v2.MapFieldMapping\", \"inputField\": { \"jsonType\": \"com.mediadriver.atlas.v2.MappedField\", \"field\": { \"jsonType\": \"com.mediadriver.atlas.java.v2.JavaField\", \"name\": \"ScreenName\", \"path\": \"User.screenName\", \"type\": \"STRING\", \"getMethod\": \"getScreenName\" }, \"fieldActions\": { \"fieldAction\": [] } }, \"outputField\": { \"jsonType\": \"com.mediadriver.atlas.v2.MappedField\", \"field\": { \"jsonType\": \"com.mediadriver.atlas.java.v2.JavaField\", \"name\": \"Title\", \"path\": \"Title\", \"type\": \"STRING\", \"setMethod\": \"setTitle\" }, \"fieldActions\": { \"fieldAction\": [] } } } ] }, \"name\": \"twitterStatusToSalesforceContact\", \"sourceUri\": \"atlas:java?className=twitter4j.Status\", \"targetUri\": \"atlas:java?className=com.redhat.ipaas.connector.salesforce.Contact\" } } "
```
Comment (rhuss) [inline dao/src/main/resources/com/redhat/ipaas/dao/deployment.json]: Is this change intended ? i.e the model class name change ? ask because it seems to be unrelated to the PR's task.

File: project-generator/src/test/resources/com/redhat/ipaas/project/converter/test-pull-push-funktion.yml [hunk 2]
```diff
@@ -4,9 +4,9 @@ flows:
...
     uri: periodic-timer:every?period=5000
   - kind: endpoint
-    uri: http-get:http://localhost:8080/hello
+    uri: http-get:?httpUri=http://localhost:8080/hello
   - kind: log
     message: Hello World! ${body}
     loggingLevel: INFO
   - kind: endpoint
-    uri: http-post:http://localhost:8080/bye
+    uri: http-post:?httpUri=http://localhost:8080/bye
```
Comment (rhuss) [inline project-generator/src/test/resources/com/redhat/ipaas/project/converter/test-pull-push-funktion.yml]: Guess this fix is required here, too. just asking, not that something slips through accidentally :)

File: dao/src/main/resources/com/redhat/ipaas/dao/deployment.json (hunk 1 above)
Comment (chirino) [inline dao/src/main/resources/com/redhat/ipaas/dao/deployment.json]: yeah don't think I intended this one.

File: project-generator/src/test/resources/com/redhat/ipaas/project/converter/test-pull-push-funktion.yml (hunk 2 above)
Comment (chirino) [inline project-generator/src/test/resources/com/redhat/ipaas/project/converter/test-pull-push-funktion.yml]: This one was.

**Comment by KurtStam** [comment]
@chirino Does it fix this then: https://github.com/redhat-ipaas/ipaas-rest/issues/261 ?
